### ISSUE TITLE
Logs Panel: Table UI - Wrap text

### ISSUE THREAD

**Issue by gtk-grafana** [grafana/grafana#78840]
For table cells with longer content (like log lines), seeing the first few characters isn't always very helpful.
Ideally we could have a setting like in the logs panel which will change the height of table rows so the content can fit the width of the table cell without truncation. 

<img width="164" alt="image" src="https://github.com/grafana/grafana/assets/109082771/6cb6b2f8-12ee-40b0-ade6-00ff5379c003">

<img width="1397" alt="image" src="https://github.com/grafana/grafana/assets/109082771/665b89bd-b3ac-41f4-a629-25583fa850ee">

Note: https://github.com/grafana/grafana/issues/25623, this has been a longstanding request for the table component, however I still think there is value in creating a new issue in the context of the logs table UI in explore, as there might be solutions we can investigate that don't involve changing the base component.

**Comment by joehacksalot** [comment]
Thanks for opening this. I came across your issue after finding the 2020 issue that looks to be abandoned while trying to figure out how to wrap the text in my table. Hope to have this capability soon.

**Comment by gtk-grafana** [comment]
We're waiting on a [new version of the table component](https://github.com/grafana/grafana/issues/94754) which will have first-class support for line wrapping.

**Comment by shiehand** [comment]
Hey @gtk-grafana our users would greatly appreciate this feature when using the Table view.
Do you have any updates on this? Thanks! 🙏

**Comment by futuarmo** [comment]
Hello! Are there any news? It's really necessary to see a lot of values of one field on screen

**Comment by matyax** [comment]
This has been implemented as part of https://github.com/grafana/grafana/issues/116688, currently behind the feature flag `logsTablePanelNG`
